helpers.lua

```diff
@@ -244,8 +244,17 @@ function helpers.sysctl_async(path_table, parse)
 
     spawn.with_line_callback("sysctl " .. path, {
         stdout = function(line)
+	    local separators = {
+	       openbsd = "=",
+	       linux = " = ",
+	       freebsd = ": "
+	    }
+
+	    local pattern = "(.+)%s(.+)"
+	    pattern = pattern:format(separators[helpers.getos()])
```

Comment:
I personally believe this is unnecessarily verbose--if the pattern fit in the `match` below within 80 columns, please uses it anonymously.

Reply:
It was. I could've written it differently. Thanks.